webapp/channels/src/components/new_channel_modal/new_channel_modal.tsx

```diff
@@ -283,16 +284,18 @@ const NewChannelModal = () => {
                     onChange={handleOnTypeChange}
                 />
                 <div className='new-channel-modal-purpose-container'>
-                    <textarea
+                    <Input
```

Comment:
I think this will have to be changed to use the new advancedtexteditor component (channels/src/components/advanced_text_editor/advanced_text_editor.tsx) the one we are using in the new channel settings modal, that could slightly change how it looks, but let's confirm with @abhijit-singh .

Reply:
Really? That seems like a lot for just a small bit of text.
As I understand this isn't used particularly frequently, so is it worth the effort?

Reply:
@pvev @abhijit-singh This is holding up other fixes. Do we have a decision on this? If not, I can pull out this change and we can merge the other fixes.

Reply:
ok, no need for now, we can revisit this later  👍 .